### DISCUSSION TITLE
Github Student benefits application getting rejected

### DISCUSSION THREAD

**Discussion by RashmiNirasha** [community/community#66521] [GitHub Education]
### Hi everyone,

Hi everyone, I keep getting this error even when I update my profile with the exact name in my student ID. Do I need to include Ms. Part as well? I'm lost. help. included a bio as well.

![image](https://github.com/community/community/assets/90468021/0345b6d9-264e-432a-8895-70e0fa9e8a63)

**Comment by Mrt009** [comment]
I think it may be due to blurry image I have also faced the same issue.   Even i tried  to scan the id card by camera live but it also did  not work.  As a last try I took the photo with better lighting condition and it works verified and got pro account within few minutes of request.
 Note: Try to capture each and every part including borders without missing any thing ,your name , grade  card expiry date should be clearly visilble.
(I have also got the same msg before although it was year back)
